### ISSUE TITLE
about the frst commit made by gerald

### ISSUE THREAD

**Issue by gracenamuli** [androidcode33/Group_210_Eve#1]
i think this is mainly about physical security of computers,l find your introduction mainly about vision.i think the vision for computer physical security is what we are supposed to be talking about here

**Comment by androidcode33** [comment]
no, coz our topic first is computer vision for physical security not physical security for computers

**Comment by Dexterobs** [comment]
ok actually the topic is a bit confusing coxz i tried to do a depth analysis but even google doesn't understand what we are looking for so i guess lets focus on what the computer world has and can do for the physical security of maybe persons and their properties which is the world in a broader picture.
